pwndbg/heap/ptmalloc.py

```diff
@@ -1408,18 +1419,18 @@ def main_arena(self):
                                 found = True
                                 break
                             tmp_arena = self.malloc_state(tmp_next)
-                            if (
-                                pwndbg.gdblib.vmmap.find(tmp_arena.get_field_address("next"))
-                                is not None
-                            ):
+                            try:
                                 tmp_next = int(tmp_arena["next"])
-                            else:
+                            except Exception:
                                 # if `&tmp_arena->next` is not valid, the linked list is broken, break this while loop and try `addr+pwndbg.gdblib.arch.ptrsize` again
```

Comment:
Just to be sure: this comment here is still valid after we switch to except, right?

Reply:
Yes, it should still be valid.